app/assets/stylesheets/article-show.scss

```diff
@@ -1,324 +1,324 @@
+        @media screen and (min-width: 1600px) {
+          font-size: 45px;
         }
       }
-      &.longest{
-        font-size:calc(1.42vw + 22px);
-        @media screen and (min-width: 1600px ){
-          font-size:42px;
+      &.longest {
+        font-size: calc(1.42vw + 22px);
+        @media screen and (min-width: 1600px) {
+          font-size: 42px;
         }
       }
     }
-    h3{
-      margin:4px auto;
-      padding:0;
-      padding:0px 3px 16px;
+    h3 {
+      margin: 4px auto;
+      padding: 0;
+      padding: 0px 3px 16px;
       font-weight: 500;
-      color:$medium-gray;
-      font-size:13.5px;
-      line-height:1.5em;
-      @media screen and ( min-width: 430px ){
-        font-size:15px;
+      color: $medium-gray;
+      font-size: 13.5px;
+      line-height: 1.5em;
+      @media screen and (min-width: 430px) {
+        font-size: 15px;
       }
 
-      .profile-pic{
-        width:28px;
-        height:28px;
-        border-radius:50px;
-        display:inline-block;
-        vertical-align:-9px;
-        margin-right:2px;
-        background:$light-gray;
-
+      .profile-pic {
+        width: 28px;
+        height: 28px;
+        border-radius: 50px;
+        display: inline-block;
+        vertical-align: -9px;
+        margin-right: 2px;
+        background: $light-gray;
       }
-      a{
-        color:$medium-gray;
-        text-decoration:none;
+      a {
+        color: $medium-gray;
+        text-decoration: none;
       }
-      .icon-img{
-        opacity:0.5;
-        width:18px;
-        height:18px;
-        vertical-align:-4px;
-        margin:0px 1px;
-        &:hover{
-          opacity:0.6;
+      .icon-img {
+        opacity: 0.5;
+        width: 18px;
+        height: 18px;
+        vertical-align: -4px;
+        margin: 0px 1px;
+        &:hover {
+          opacity: 0.6;
         }
       }
-      .published-at{
-        margin-right:0px;
-        display:inline-block;
+      .published-at {
+        margin-right: 0px;
+        display: inline-block;
       }
       .posted-date-inline {
-        display:inline-block;
+        display: inline-block;
       }
-      .action-space{
-        a{
-          display:inline-block;
-          background:$green;
-          color:$black;
-          padding:2px 8px 3px;
-          border-radius:3px;
-          line-height:1.1em;
+      .action-space {
+        a {
+          display: inline-block;
+          background: $green;
+          color: $black;
+          padding: 2px 8px 3px;
+          border-radius: 3px;
+          line-height: 1.1em;
           font-family: $helvetica-condensed;
-          font-stretch:condensed;
-          @media screen and ( max-width: 376px ){
-            padding:2px 3px 3px;
-            margin-left:3px;
-            margin-left:5px;
-            margin-top: 5px;
-            .post-word{
-              display:none;
+          font-stretch: condensed;
+          @media screen and (max-width: 340px) {
```

Comment:
This should remain as 376px -- we changed it recently so you might have needed to rebase?

Reply:
Haven't done a rebase before, I've tried the following in the feature branch but it doesn't restore the css that's changed or missing.

```
git rebase master
```

Reply:
Shoot. I forgot that rebasing isn't the best idea after the PR's been reviewed. Can you update your master branch and then fix conflicts via `git merge master`?

Reply:
I've done a `git merge master` after updating my master branch,